src/blog/2024/01/dashboard-2-ga.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Dashboard 2.0 is Generally Available!
+subtitle: This week sees our first major version release of Node-RED Dashboard 2.0!
+description: This week sees our first major version release of Node-RED Dashboard 2.0!
+date: 2024-01-16
+authors: ["joe-pavitt"]
+image: "/blog/2023/12/images/tile-dashboard-0-10-0.png"
+tags:
+    - posts
+    - releases
+    - community
+    - changelog
+    - dashboard
+---
+
+Back in [June 2023](/blog/2023/06/dashboard-announcement/) we announced that FlowFuse would be investing time into building out the next generation of Node-RED Dashboard, the incredibly popular UI framework for Node-RED.
+
+We followed this up with the [first release](/blog/2023/07/dashboard-0-1-release/) (`0.0.1`) in July, just one month later, and today, we are pleased to announce that we have reached a major milestone in this journey, with the release of our first major version (`1.0.0`) of Node-RED Dashboard 2.0.
+
+<!--more-->
+
+With Node-RED Dashboard 2.0, we have re-built the original Node-RED Dashboard from the ground up. We're now VueJS-based, completely responsive down to mobile and have made many quality of life improvements across the board to the existing widget collection, as well as adding a few new ones too.
+
+We've shared plenty of updates sicne we started, detailing the feature parity with the original Node-RED Dashboard, as well as some of the new widgets and features we've added to the new Dashboard, such as Markdown, Mermaid Charts and new Layout Options, you can read more about those here:
```

Comment:
```suggestion
We've shared plenty of updates since we started, detailing the feature parity with the original Node-RED Dashboard, as well as some of the new widgets and features we've added to the new Dashboard, such as Markdown, Mermaid Charts and new Layout Options, you can read more about those here:
```